### PR TITLE
added tests for serialization.py

### PR DESCRIPTION
**Pull request by cat-bug**

### CODE REVIEW

File: tests/test_serialization.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        serialization.write_list(value, self.filehandler, func_wr)
+        self.filehandler.flush()
+        self.filehandler.seek(0)
+        actual = serialization.read_list(self.filehandler, func_rd)
+        assert actual == value
+
+    @pytest.mark.parametrize(
+        "value",
+        [
+            ([True, False]),
```
Comment (andrewprzh) [inline tests/test_serialization.py:70]: could you add an empty list test?

File: tests/test_serialization.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        ]
+    )
+    def test_write_read_positive(self, value, func_wr, func_rd):
+        func_wr(value, self.filehandler)
+        self.filehandler.flush()
+        self.filehandler.seek(0)
+        actual = func_rd(self.filehandler)
+        assert actual == value
+
+    @pytest.mark.parametrize(
```
Comment (andrewprzh) [inline tests/test_serialization.py:51]: could you add 1-2 empty lists here too?

**Review by andrewprzh — CHANGES_REQUESTED** [review]
all good! just add empty list tests please

File: tests/test_serialization.py (hunk 2 above)
Comment (cat-bug) [reply tests/test_serialization.py:54]: done

**Comment by cat-bug** [comment]
coverage of `serialization.py` is increased from 39% to 97%

**Review by andrewprzh — APPROVED** [review]

**Pull request merged**